swift/Podfile.lock

```diff
@@ -17,4 +17,4 @@ SPEC CHECKSUMS:
 
 PODFILE CHECKSUM: 4382612b240ed6d49e9a3b9bbb107c581e72b8ea
 
-COCOAPODS: 1.9.3
+COCOAPODS: 1.9.1
```

Comment:
Can you revert this file please?  Probably unintended.

Reply:
@catenocrypt Fixed this